### DISCUSSION TITLE
Eror

### DISCUSSION THREAD

**Discussion by Crisatian** [helium/miner#1660] [Issues]
failed to get peer book. Post "http://%2Fvar%2Frun%2Fdocker.sock/v1.29/containers/fd674192cbe98c3bd673d4f4bab6021577080bef8660f12855ed171e4052df69/exec": context deadline exceeded

**Comment by meowshka** [comment]
@Crisatian who's your hotspot manufacturer?

**Reply by Crisatian** [reply]
Bobcat300

**Reply by fadiemman** [reply]
I am also getting the same error.
